src/overlays/actors/ovl_En_Geg/z_en_geg.c

```diff
@@ -97,87 +124,955 @@ static DamageTable D_80BB3ED4 = {
+        }
+    } else if ((sp27 == 5) && func_80147624(globalCtx)) {
+        if (this->unk_496 == 0xD67) {
+            globalCtx->msgCtx.unk11F22 = 0x43;
+            globalCtx->msgCtx.unk12023 = 4;
+            this->unk_49A = this->unk_49C[4];
+            this->actionFunc = func_80BB2520;
+        } else {
+            this->unk_496 = func_80BB16D0(this);
+            func_80151938(globalCtx, this->unk_496);
+        }
+    }
+}
+
+void func_80BB2A54(EnGeg* this, GlobalContext* globalCtx) {
+    if ((func_80152498(&globalCtx->msgCtx) == 5) && func_80147624(globalCtx)) {
+        if (this->unk_496 == 0xD65) {
+            ActorCutscene_Stop(this->unk_498);
+            this->unk_230 &= ~0x10;
+            this->unk_244 = 65;
+            globalCtx->msgCtx.unk11F22 = 0x43;
+            globalCtx->msgCtx.unk12023 = 4;
+            this->actionFunc = func_80BB347C;
+        } else {
+            this->unk_496 = func_80BB16D0(this);
+            func_801518B0(globalCtx, this->unk_496, &this->actor);
+        }
+    }
+}
+
+void func_80BB2B1C(EnGeg* this, GlobalContext* globalCtx) {
+    Vec3f sp74;
+    f32 temp_f20;
+    s16 i;
+    f32 sp68;
+
+    this->actor.child->world.pos = this->unk_4B4;
+    this->actor.child->shape.rot = this->actor.shape.rot;
+
+    if (ActorCutscene_GetCurrentIndex() != this->unk_49C[4]) {
+        if (ActorCutscene_GetCanPlayNext(this->unk_498)) {
+            gSaveContext.weekEventReg[37] |= 8;
+            if (this->actor.child != NULL) {
+                Actor_MarkForDeath(this->actor.child);
+            }
+            this->unk_230 |= 0x10;
+            ActorCutscene_StartAndSetFlag(this->unk_498, &this->actor);
+            this->unk_496 = 0xD68;
+            func_80151938(globalCtx, this->unk_496);
+            this->unk_248 = Object_GetIndex(&globalCtx->objectCtx, OBJECT_TAISOU);
+            if (this->unk_248 >= 0) {
+                this->unk_4AC = 13;
+                func_80BB2020(this, globalCtx);
+            }
+            this->actionFunc = func_80BB27D4;
+        } else {
+            if (ActorCutscene_GetCurrentIndex() == 0x7C) {
+                ActorCutscene_Stop(0x7C);
+            }
+            ActorCutscene_SetIntentToPlay(this->unk_498);
+        }
+    } else {
+        temp_f20 = this->unk_4E0 * 0.005f;
+        sp68 = this->unk_4E0 * 0.07f;
+
+        sp74.x = Rand_Centered() * temp_f20;
+        sp74.y = Rand_Centered() * temp_f20;
+        sp74.z = Rand_Centered() * temp_f20;
+
+        this->actor.child->scale.x *= 0.98f;
+        this->actor.child->scale.y *= 0.98f;
+        this->actor.child->scale.z *= 0.98f;
+
+        if (this->unk_4E0 > 70) {
+            for (i = 0; i < ARRAY_COUNT(this->unk_4C0); i++) {
+                this->unk_4C0[i].x *= temp_f20;
+                this->unk_4C0[i].y *= temp_f20;
+                this->unk_4C0[i].z *= temp_f20;
+                EffectSsHahen_Spawn(globalCtx, &this->unk_4B4, &this->unk_4C0[i], &sp74, 1, sp68, 1, 15, D_0406AB30);
```

Comment:
`D_0406AB30` is from gameplay_keep. Should be able to include and use the symbol.

Reply:
Looks like we already it.